components/builder-depot/src/server.rs

```diff
@@ -1103,6 +1103,30 @@ fn get_schedule(req: &mut Request) -> IronResult<Response> {
     }
 }
 
+// TODO (SA) This is an experiemental dev-only function for now
+fn abort_schedule(req: &mut Request) -> IronResult<Response> {
+    let group_id = {
```

Comment:
This should be an authenticated request even if it is experimental so people don’t cancel each other’s jobs :)

Reply:
ah yeah - thanks for the catch!

Reply:
I've made it require  worker privilege for now - getting the right access check is one of the things that will need to be done right later.